lib/board.rb

```diff
@@ -0,0 +1,44 @@
+class Board
+  attr_accessor :cells
+
+  def initialize
+    reset!
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(input)
+    cells[input.to_i-1]
+  end
+
+  def full?
+    cells.all?{ |character| character == "X" || character == "O"}
```

Comment:
Nice job using the `all?` iterator. You could make this even more DRY by checking that all spaces are not empty (rather than whether they contain an X or O) — something like this:

`cells.all? {|cell| cell != " "}`